package.json

```diff
@@ -1 +1,5 @@
-{}
\ No newline at end of file
+{
```

Comment:
wrong package.json, bri-3/package.json is the one to install this

Reply:
fixed